zerver/actions/realm_settings.py

```diff
@@ -104,6 +105,43 @@ def do_set_realm_property(
         update_users_in_full_members_system_group(realm, acting_user=acting_user)
 
 
+@transaction.atomic(durable=True)
+def do_set_realm_permission_group_settings(
```

Comment:
`do_change_realm_permission_group_setting` (singular, and using `change`) feels clearer to me. We're changing a single group setting.

The reason I prefer "change" over "set" is that we're moving a pointer (the group ID), effectively, rather than changing a value.